docker/ros/kinetic/v4r/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM shadowrobot/build-tools:xenial-kinetic
+
+MAINTAINER "Shadow Robot's Software Team <software@shadowrobot.com>"
+
+LABEL Description="Image for using sr_recognizer package" Version="1.0"
+
+RUN curl -s https://raw.githubusercontent.com/shadow-robot/sr-build-tools/master/docker/utils/v4r_and_openni_kinetic.sh | bash && \
```

Comment:
please have a look how this URL is taking into account branch of the build tools https://github.com/shadow-robot/sr-build-tools/blob/master/docker/ros/kinetic/ide/Dockerfile#L23 (sometimes it is needed for debugging)